libs/core/src/lib/wizard/wizard-step-indicator/wizard-step-indicator.component.html

```diff
@@ -1,3 +1,4 @@
-<span class="fd-wizard__step-indicator" [ngClass]="glyph ? 'sap-icon--' + glyph : ''">
+<span class="fd-wizard__step-indicator">
+    <i class="fd-wizard__icon" [ngClass]="glyph ? 'sap-icon--' + glyph : ''"></i>
```

Comment:
We shouldn't render this icon when we don't have a `glyph` specified.
Also could you please use `<fd-icon>` for that?